### PR TITLE
Added WoC and ADL to trackable NMs

### PR DESCRIPTION
**Pull request by xurion**
* Adjusted addon to allow tracking of NMs that use regular items as pops.
* Allowed tracking of NMs with spaces in their name.

### CODE REVIEW

File: addons/EmpyPopTracker/EmpyPopTracker.lua [hunk 1]
```diff
@@ -167,16 +167,22 @@ end
...
-            if not has_pop_ki then
-                info.has_all_kis = false
+            if pop_item_data.type == 'key item' then
+                has_pop = owns_key_item(pop_item_data.id, key_items)
+            elseif pop_item_data.type == 'item' then
+                has_pop = owns_item(pop_item_data.id, items)
+            end
+
+            if not has_pop then
+                info.has_all_pops = false
```
Comment (z16) [inline addons/EmpyPopTracker/EmpyPopTracker.lua]: The entire function body can be replaced with:
```lua
return {
    has_all_pops = not nm.pops or nm.pops:all(function(item)
        return pop.type == 'item' and owns_item(pop.id, items) or owns_key_item(pop.id, key_items)
    end)
    text = generate_text(nm, key_items, items, 1)
}
```

If it complains about `all` being `nil` then you need to do `T(nm.pops):all`.

Adjust as you see fit for readability.

**Review by z16 — APPROVED** [review]
Minor remark, but looks good. Let me know if you wanna adjust it somehow or if you want me to merge it as it is.

File: addons/EmpyPopTracker/EmpyPopTracker.lua (hunk 1 above)
Comment (xurion) [reply addons/EmpyPopTracker/EmpyPopTracker.lua]: Nice, I like it and I have just pushed the change. Thanks for the speedy review!

**Comment by RubenatorX** [comment]
Is a name change in order? :X WoC isn't exactly an EmpyPop xD

**Comment by z16** [comment]
Eh :D

**Comment by xurion** [comment]
I didn't even want to contemplate what that would mean for backwards compatibility XD In hindsight, I was too quick to choose the name

**Pull request merged**